src/gui/search/searchjobwidget.cpp

```diff
@@ -315,6 +315,7 @@ void SearchJobWidget::updateFilter()
 void SearchJobWidget::fillFilterComboBoxes()
 {
     using Utils::Misc::SizeUnit;
+    using Utils::Misc::unitString;
```

Comment:
please add blank line.

Reply:
Not sure where did you mean it, added below.

Reply:
Yes, thank you!